DOCUMENTATION.md

```diff
@@ -439,32 +439,32 @@ You can only get the status of messages that are 7 days old or newer.
+|Failed|This covers all failure statuses:<br>- `permanent-failure` - "The provider could not deliver the message because the email address or phone number was wrong. You should remove these email addresses or phone numbers from your database. You’ll still be charged for text messages to numbers that do not exist."<br>- `temporary-failure` - "The provider could not deliver the message. This can happen when the recipient's inbox is full, or their phone is off. You can try to send the message again."<br>- `technical-failure` - "Your message was not sent because there was a problem between Notify and the provider.<br>You’ll have to try sending your messages again. You will not be charged for text messages that are affected by a technical failure."|
 
 ## Status - text only
 
 |Status|Information|
 |:---|:---|
-|Pending|GOV.UK Notify received a callback from the provider but the device has not yet responded. Another callback from the provider determines the final status of the notification.|
-|Sent|The text message was delivered internationally. This only applies to text messages sent to non-UK phone numbers. GOV.UK Notify may not receive additional status updates depending on the recipient's country and telecoms provider.|
+|Pending|GOV.UK Notify is waiting for more delivery information.<br>GOV.UK Notify received a callback from the provider but the recipient's device has not yet responded. Another callback from the provider determines the final status of the notification.|
+|Sent|The message was sent to an international number. The mobile networks in some countries do not provide any more delivery information.|
 
 ## Status - letter
 
 |Status|information|
 |:---|:---|
 |Failed|The only failure status that applies to letters is `technical-failure`. GOV.UK Notify had an unexpected error while sending to our printing provider.|
-|Accepted|GOV.UK Notify is printing and posting the letter.|
-|Received|The provider has received the letter to deliver.|
+|Accepted|GOV.UK Notify has placed the letter in a queue, ready to be sent to the provider.|
+|Received|GOV.UK Notify has sent the letter to the provider. The provider has printed and dispatched the letter.|
 
 ## Status - precompiled letter
 
 |Status|information|
 |:---|:---|
-|Pending virus check|GOV.UK Notify virus scan of the precompiled letter file is not yet complete.|
-|Virus scan failed|GOV.UK Notify virus scan has identified a potential virus in the precompiled letter file.|
+|Pending virus check|GOV.UK Notify has not completed a virus scan of the pre-compiled letter file.|
```

Comment:
I have a question for our developers about lines 466 and 467.
The descriptions are fine (I think), but I'd like to know if we can make the statuses consistent, so we don't say virus check *and* virus scan.